src/main/java/frc/robot/auton/AutonIntake.java

```diff
@@ -10,4 +10,10 @@ public boolean isDone() {
 
     @Override
     public void initiate() {}
+
+    @Override
+    public void shutdown() {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'shutDown'");
+    }
```

Comment:
AutonIntake isn't set up properly right now, so we don't have anything to shutdown